CMakeLists.txt

```diff
@@ -48,6 +48,8 @@
 # Visual Studio 2017 requires 3.7.
 cmake_minimum_required(VERSION 3.7)
 
+message("CMake version ${CMAKE_VERSION}")
```

Comment:
This should have the `STATUS` type: `message(STATUS ...`).

Reply:
Done.